sktime/forecasting/statsmodels_arima.py

```diff
@@ -0,0 +1,316 @@
+import pandas as pd
+
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+
+
+class StatsModelsARIMA(_StatsModelsAdapter):
+    """ARIMA forecaster.
+
+    Direct interface for `statsmodels.tsa.arima.model.ARIMA`.
+
+    Parameters
+    ----------
+    order : tuple, optional The (p, d, q) order of the model for the autoregressive,
+        differences, and moving average components. `d` is always an integer, while `p`
+        and `q` may either be integers or lists of integers.
+
+    seasonal_order : tuple, optional The (P, D, Q, s) order of the seasonal component
+        of the model for the AR parameters, differences, MA parameters, and periodicity.
+        Default is (0, 0, 0, 0). `D` and `s` are always integers, while `P` and `Q` may
+        either be integers or lists of positive integers.
+
+    trend : str{'n', 'c', 't', 'ct'} or iterable, optional Parameter controlling the
+        deterministic trend. Can be specified as a string where 'c' indicates a constant
+        term, 't' indicates a linear trend in time, and 'ct' includes both. Can also be
+        specified as an iterable defining a polynomial, as in numpy.poly1d, where
+        [1,1,0,1] would denote :math:`a + bt + ct^3`. Default is 'c' for models without
+        integration, and no trend for models with integration. Note that all trend terms
+        are included in the model as exogenous regressors, which differs from how trends
+        are included in SARIMAX models. See the Notes section for a precise definition
+        of the treatment of trend terms.
+
+    enforce_stationarity : bool, optional Whether to require the autoregressive
+        parameters to correspond to a stationarity process.
+
+    enforce_invertibility : bool, optional Whether to require the moving average
+        parameters to correspond to an invertible process.
+
+    concentrate_scale : bool, optional Whether to concentrate the scale (variance of
+        the error term) out of the likelihood. This reduces the number of parameters by
+        one. This is only applicable when considering estimation by numerical maximum
+        likelihood.
+
+    trend_offset : int, optional The offset at which to start time trend values.
+        Default is 1, so that if `trend='t'` the trend is equal to 1, 2, …,
+        nobs. Typically, is only set when the model created by extending a previous
+         dataset.
+
+    dates : array_like of datetime, optional If no index is given by `endog` or
+        `exog`, an array-like object of datetime objects can be provided.
+
+    freq : str, optional If no index is given by `endog` or `exog`, the frequency of
+        the time-series may be specified here as a Pandas offset or offset string.
+
+    missing : str, optional Available options are 'none', 'drop', and 'raise'. If
+        'none', no nan checking is done. If 'drop', any observations with nans are
+        dropped. If 'raise', an error is raised. Default is 'none'.
```

Comment:
I think it's customary to only show the type hint and parameter kind (optional or not) in first line, and rest to show in the next ones. Like you did below.

You can even copy paste from here exactly as it is :)

https://www.statsmodels.org/stable/_modules/statsmodels/tsa/arima/model.html#ARIMA

Reply:
Sure, will reformat that!